src/components/MovieInfo.tsx

```diff
@@ -15,25 +15,31 @@ export type DataChildren = {
 };
 
 const MovieInfo: React.FC<MovieInfoProps> = ({ data }) => {
-
   return (
     <div className="container">
       <div className="row justify-content-between align-items-center">
-        {data.map((item, i) => {
-          const { Title, Year, Type, Poster } = item
-          const icon = Type === 'movie' ? <BiCameraMovie /> : <PiTelevisionSimple />
+        {data.map(({ Title, Year, Type, Poster }, i) => {
           return (
             <div key={i} className="col-4">
               <div className="card text-white bg-dark mb-3 m-2">
```

Comment:
Please add an inline max height and regular height to this section to keep consistency among all cards:
```suggestion
<div style={{height: '650px', maxHeight: '700px'}} className="card text-white bg-dark mb-3 m-2">
```

Reply:
```suggestion
              
``` <div key={i} className="col-4 my-2">
              <div
                style={{ height: "100%", maxHeight: "750px" }}
                className="card text-white bg-dark mb-3 m-2"
              >
                <img 
                  style={{height: "80%"}}
                  className="card-img-top" 
                  src={Poster} 
                  alt={Title} />
                <div className="card-body d-flex flex-column justify-content-center">

Reply:
May I suggest we do these changes in this section? With the first suggestion the text from the card body element was overflowing out of the container. 

With the second suggestion the images are the same size and the card body elements are centered using flex. Let me know what you think!

Reply:
Proceed